package.json

```diff
@@ -16,13 +18,12 @@
     "jquery": "^3.5.1",
     "popper.js": "^1.16.1",
     "ramda": "^0.27.1",
+    "serve": "^11.3.2",
```

Comment:
Why do you add this dependency?

Reply:
i might have used npm install locally...

Reply:
but i didn't add any package on purpose

Reply:
now i understand this shit XD i didn't write "npm run serve" correctly and add the "serve" to the package... lolololol